pages/docs/concepts/asyncapi-document/structure.md

```diff
@@ -0,0 +1,432 @@
+
+The `info` object includes properties such as:
+
+- `title`: The title of the API.
+- `version`: The version of the API.
+- `description`: A brief description explaining the purpose and features of the API.
+- `termsOfService`: The URL or document specifying the terms of service for using the API.
+- `contact`: Contact information for the owner or maintainer of the API, including name, email, and URL.
+- `license`: Information about the license under which the API is provided, including name and URL.
+- `tags`: A list of tags for application API documentation control. Tags can be used for logical grouping of applications.
+- `externalDocs`: Additional external documentation of the exposed API.
+
+Here's a visual representation of the `info` object and its properties:
+```mermaid
+graph LR
+  B(info)
+  C(title)
+  D(version)
+  E(description)
+  F(termsOfService)
+  G(contact)
+  J(license)
+  M(tags)
+  P((externalDocs))
+
+  B --> C
+  B --> D
+  B --> E
+  B --> F
+  B --> G
+  B --> J
+  B --> M
+  B --> P
+```
+
+Below is a example of the `info` object in the AsyncAPI document:
+```yaml
+asyncapi: 3.0.0
+info:
+  title: My Event-Driven API
+  version: 1.0.0
+  description: This API provides real-time event streaming capabilities.
+  termsOfService: https://example.com/terms-of-service
+  contact:
+    name: Rohit
+    email: rohit@asyncapi.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  tags:
+    - name: Events
+      description: APIs related to event streaming
+    - name: Authentication
+      description: APIs for authentication and authorization
+  externalDocs:
+    description: Additional documentation 
+    url: https://example.com/docs
+```
+
+### `Servers` object
+The `servers` object in an AsyncAPI document is a map of `server` objects that defines the network endpoints or brokers that applications can connect to for exchanging messages. It specifies the details necessary to establish a connection, such as the protocol, host, port, and additional connection options. The `server` object is used to capture details such as URIs, protocols and security configuration. 
+
+The purpose of the `servers` object is to provide the necessary information for clients to connect to the message broker or server and participate in the message exchange or listen to the events. By defining multiple servers, the AsyncAPI document can accommodate different environments or deployment scenarios, such as production, staging, or development.
+
+The `servers` object typically includes the following properties(few are mandatory, few are optional) for each server:
```

Comment:
```suggestion
The `servers` object typically includes the following properties for each server:
```

I don't know how I feel about this extra clarification. We're never speaking about fields being mandatory or optional so I don't think we should here either. For that, we have the reference, right?

Reply:
About this, Yes, we have reference for that but I just thought maybe we need to put out the extra information JIC.